StRoot/StFttRawHitMaker/StFttRawHitMaker.h

```diff
@@ -0,0 +1,63 @@
+/***************************************************************************
+ *
+ * StFttRawHitMaker.h
+ *
+ * Author: jdb 2021
+ ***************************************************************************
+ *
+ * Description: StFttRawHitMaker - class to fill the StEvent from DAQ reader
+ *
+ ***************************************************************************/
+#ifndef STFTTRAWHITMAKER_H
+#define STFTTRAWHITMAKER_H
+
+#include "StRTSBaseMaker.h"
+
+#include <vector>
+
+#ifndef __CINT__
+// copied from RTS ... trying to include directly fails
```

Comment:
What is the error message?

Reply:
It is likely due to the inclusion of "sys/types.h" for u_char and u_short. I saw rootcint choking on it in https://github.com/star-bnl/star-sw/pull/101#issuecomment-899818930

Reply:
I've never heard of u_char or u_short. The ISO C standard defines `<stdint.h>` that comes with (u)int?_t, I wonder if that would be more portable.

Reply:
I think it should be fine to change those types. However, this is a direct copy from RTS code.

Reply:
I will submit some PRs to address the types change. However, can we, please, update the comment to mention which header was supposed to be included and what is the error message that prevents us from using it?

Reply:
Works for me
```diff
diff --git a/StRoot/StFttRawHitMaker/StFttRawHitMaker.h b/StRoot/StFttRawHitMaker/StFttRawHitMaker.h
index 9eb42367d7..dcb6df41ea 100644
--- a/StRoot/StFttRawHitMaker/StFttRawHitMaker.h
+++ b/StRoot/StFttRawHitMaker/StFttRawHitMaker.h
@@ -15,18 +15,10 @@
 
 #include <vector>
 
-#ifndef __CINT__
-// copied from RTS ... trying to include directly fails
-struct stgc_vmm_t {
-    u_char feb_vmm ;    
-    u_char ch ;
-    u_short adc ;
-    u_short bcid ;
-    short tb ;  // from the trigger
-} ;
-#endif
 #ifdef __CINT__
 struct stgc_vmm_t;
+#else
+#include <RTS/src/DAQ_STGC/daq_stgc.h>
 #endif
 
 class StEvent;
```

Reply:
Ah great. 
I must have remembered wrong. I'll update this 👍